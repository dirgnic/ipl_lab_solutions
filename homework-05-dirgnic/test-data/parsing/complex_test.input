let x = 5;
let y = 10;
if (x + y == 15) { return 1; } else { return 0; }
