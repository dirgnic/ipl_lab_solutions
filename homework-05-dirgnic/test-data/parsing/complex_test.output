4:1:
  |
4 | <empty line>
  | ^
unexpected end of input
expecting "if", "let", or "return"
