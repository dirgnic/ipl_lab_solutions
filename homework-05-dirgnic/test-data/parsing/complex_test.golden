Seq [Let "x" (Num 5) (Res (Num 0)), Let "y" (Num 10) (Res (Num 0))] 
(Ite (Sma (Add (Var "x") (Var "y")) (Num 15)) (Res (Num 1)) (Res (Num 0)))
